### PR TITLE
fix(repo): not all changes picked up by mutation job

### PR DESCRIPTION
**Pull request by jogold**
Use `git add --all` otherwise changes in folders outside the working directory (like `docs`) are not picked up.

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Monada Contribution License](https://docs.winglang.io/terms-and-policies/contribution-license.html)*.

### CODE REVIEW

File: .github/workflows/wingsdk-mutation.yml [hunk 1]
```diff
@@ -61,7 +61,7 @@ jobs:
         if: steps.token-check.outputs.HAS_TOKEN == 'true'
         name: Find mutations
         run: |-
-          git add .
+          git add -u
```
Comment (jogold) [inline .github/workflows/wingsdk-mutation.yml:64]: This doesn't cover new files, not sure it should.

Comment (jogold) [reply .github/workflows/wingsdk-mutation.yml:64]: Maybe `-all` is better

**Review by Chriscbr — APPROVED** [review]

File: .github/workflows/wingsdk-mutation.yml (hunk 1 above)
Comment (Chriscbr) [reply .github/workflows/wingsdk-mutation.yml:64]: Introducing new snapshot files would be a valid use case for `--all`, let's try it

File: .github/workflows/wingsdk-mutation.yml [hunk 2]
```diff
@@ -61,7 +61,7 @@ jobs:
         if: steps.token-check.outputs.HAS_TOKEN == 'true'
         name: Find mutations
         run: |-
-          git add .
+          git add -all
```
Comment (jogold) [inline .github/workflows/wingsdk-mutation.yml:64]:
```suggestion
          git add --all
```

**Pull request merged**